### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### What does this PR do?

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/traefik/commit/fda566abba7f5f35e945bf471fc16b1848bbc9f3#commitcomment-67118194

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/traefik/commit/4a2ede6f1b7ec6148fb1eb181fa99dc8bd774811

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### Motivation

Spelling errors can hide bugs, make it harder to search for things, and generally tarnish products

### More

- [ ] Added/updated tests
- [ ] Added/updated documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1783,7 +1783,7 @@ Same changelog as v2.0.3.
 - fix: remove extra backtick from routers docs ([#5572](https://github.com/traefik/traefik/pull/5572) by [serpi90](https://github.com/serpi90))
 - document providersThrottleDuration ([#5519](https://github.com/traefik/traefik/pull/5519) by [mpl](https://github.com/mpl))
 - Add a response forwarding section to the service documentation  ([#5517](https://github.com/traefik/traefik/pull/5517) by [jbdoumenjou](https://github.com/jbdoumenjou))
-- Change instances of &#34;dymanic&#34; to &#34;dynamic&#34; ([#5504](https://github.com/traefik/traefik/pull/5504) by [dat-gitto-kid](https://github.com/dat-gitto-kid))
+- Change instances of &#34;dynamic&#34; to &#34;dynamic&#34; ([#5504](https://github.com/traefik/traefik/pull/5504) by [dat-gitto-kid](https://github.com/dat-gitto-kid))
```
Comment (jsoref) [inline CHANGELOG.md:1786]: Oops, I meant to skip this one...

That said, not everyone likes making changes to changelogs... If this project wants to keep these items, I can easily drop any individual files (or changes).

File: docs/content/user-guides/marathon.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ There are two mitigation efforts:
 The Marathon health check makes sure that applications once deemed dysfunctional are being rescheduled to different slaves.
 However, they might take a while to get triggered and the follow-up processes to complete.
 
-For that reason, the Treafik health check provides an additional check that responds more rapidly and does not require a configuration reload to happen.
+For that reason, the Traefik health check provides an additional check that responds more rapidly and does not require a configuration reload to happen.
```
Comment (jsoref) [inline docs/content/user-guides/marathon.md:96]: Brand

File: pkg/middlewares/tracing/wrapper.go [hunk 3]
```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/traefik/traefik/v2/pkg/tracing"
 )
 
-// Tracable embeds tracing information.
-type Tracable interface {
+// Traceable embeds tracing information.
+type Traceable interface {
```
Comment (jsoref) [inline pkg/middlewares/tracing/wrapper.go:14]: It's possible this is a public interface and thus can't be changed, it's trivial to drop such things.

File: pkg/provider/kubernetes/crd/fixtures/tcp/with_different_services_ns.yml [hunk 4]
```diff
@@ -27,5 +27,5 @@ spec:
       weight: 4
     # with unknown namespace
     - name: whoamitcp
-      namespace: unknwonns
+      namespace: unknowns
```
Comment (jsoref) [inline pkg/provider/kubernetes/crd/fixtures/tcp/with_different_services_ns.yml:30]: Perhaps this is a special namespace, but it seems more like a typo

File: pkg/provider/kubernetes/gateway/fixtures/httproute/gatewayclass_with_unknown_controller.yml [hunk 5]
```diff
@@ -4,7 +4,7 @@ apiVersion: gateway.networking.k8s.io/v1alpha2
 metadata:
   name: my-gateway-class
 spec:
-  controllerName: unkown.io/gateway-controller
+  controllerName: unknown.io/gateway-controller
```
Comment (jsoref) [inline pkg/provider/kubernetes/gateway/fixtures/httproute/gatewayclass_with_unknown_controller.yml:7]: In principle this is a violation of DNS namespace. Instead one should use an `example` domain or something owned by traefik.

**Review by jsoref — COMMENTED** [review]
Most corrections were selected automatically by Google Sheets.

All mistakes (especially the failure to drop that first item) are my fault.

**Review by tomMoulard — APPROVED** [review]
LGTM :ok_hand:

**Review by ldez — APPROVED** [review]
Thank you :+1:

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Pull request merged**